package.json

```diff
@@ -27,7 +27,7 @@
     "test:format": "prettier . -c",
     "test:unit": "vitest run unit",
     "test:integration": "vitest run integration",
-    "test:e2e": "cross-env E2E_TIMEOUT=600000 NODE_ENV=test RUNNING_IN_SPECTRON=true mocha --reporter mocha-multi-reporters --reporter-options configFile=./test/mochareporters.json ./test/e2e",
+    "test:e2e": "echo 'Currently not functional'",
```

Comment:
should we leave a TODO for these?

Reply:
also what else need to be done to resurrect e2e tests?

Reply:
Added a TODO in the script itself. As for getting them to work, well... I guess the first step is picking a framework, where my choices would be WDIO or Playwright. Then comes the part of figuring out how to integrate it, and I guess once that's done, the suite can be extended further.